dev/release/utils-prepare.sh

```diff
@@ -121,7 +139,7 @@ update_versions() {
 
   sed -i.bak -E \
     -e "s/^version = \".+\"/version = \"${rust_version}\"/" \
-    -e "s/^adbc_core = { path = \".\/core\", version = \".+\"/adbc_core = { path = \".\/core\", version = \"${rust_version}\"/" \
+    -e "s/^adbc_core = \{ path = \".\/core\", version = \".+\"/adbc_core = { path = \".\/core\", version = \"${rust_version}\"/" \
```

Comment:
It seems that `{` is a special character in "sed (GNU sed) 4.9".
I need this to run this on local.